InstallationGuide.md

```diff
@@ -1,50 +1,39 @@
 # Installation Guide
-## Product DFT
-Install from the command line:
 
-docker container run -d --name [conatainer_name] ghcr.io/catenax-ng/tx-dft-backend:[tag]
+It is necessary to inject the environment variables, credentials and URLs that can be found on application.properties file.
+please refer configuration section from here [README.md](README.md)
 
+### RUN SDE backend in ArgoCD 
+ We have helm chart available for ArgoCD deployment. In deployment, if don't specified specific version, the latest version on main is automatically picked up by ArgoCD and deployed to the environment using Helm charts.
```

Comment:
Your helm chart should not necessarily just use the latest. You could easily combine the app version and chart version to always pull the version you released.

You will need to control the version anyway for proper testing

Reply:
Re-corrected sentence.